### DISCUSSION TITLE
Your account has been flagged.

### DISCUSSION THREAD

**Discussion by 9abour** [community/community#119587] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

My account has been flagged, and I don't know the reason for that.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by paulospx** [comment]
Having the same issue, I would like to express my frustration and concern about my account being flagged. 
I've been trying to get to the bottom of this issue for months now, but unfortunately, I've received no response or resolution. 
It's been incredibly frustrating and has hindered my ability to work effectively.

As a loyal githuber, I expect a certain level of service and support. It's unacceptable that I've been ignored for so long. I'm hoping someone can finally take a look at my account and provide some clarity on why it's been flagged. I'd appreciate it if someone could reach out to me and provide a solution to this issue.

Sincerely
Paulo
